### DISCUSSION TITLE
Type error: Type '[IconPrefix, IconName]' is not assignable to type 'IconProp'.

### DISCUSSION THREAD

**Discussion by c-jeremy** [spencerwooo/onedrive-vercel-index#399] [Q&A] [answered]
While trying to redeploy onedrive-vercel-index just a few minutes ago, Vercel returned error message shown below.

> Failed to compile.

22:58:15 |  
22:58:15 | ./components/FileListing.tsx:78:22
22:58:15 | Type error: Type '[IconPrefix, IconName]' is not assignable to type 'IconProp'.
22:58:15 |  
22:58:15 | 76 \|     <span>{emoji ? emoji[0] : '📁'}</span>
22:58:15 | 77 \|   ) : (
22:58:15 | > 78 \|     <FontAwesomeIcon icon={child.file ? getFileIcon(child.name, { video: Boolean(child.video) }) : ['far', 'folder']} />
22:58:15 | \|                      ^
22:58:15 | 79 \|   )
22:58:15 | 80 \| }
22:58:15 | 81 \|

`

I am sure I forked the latest commit. Is it possibly a change of FontAwesome?

**Comment by myl7** [comment, marked as answer]
See discussion #374 , you may need to change the build and install command in Vercel console.

**Comment by myl7** [comment]
@spencerwooo How about pinning the QA to Announcements of Discussions?

**Comment by c-jeremy** [comment]
> @spencerwooo How about pinning the QA to Announcements of Discussions?

Thanks a million. For the pinning thing, I myself see that as a great idea.🤣
